diffengine/__init__.py

```diff
@@ -401,6 +402,11 @@ def setup_db():
     db.init(db_file)
     db.connect()
     db.create_tables([Feed, Entry, FeedEntry, EntryVersion, Diff], safe=True)
+    try:
```

Comment:
This is awesome -- I didn't know about migrations for sqlite. This should just run one time at startup right?

Reply:
Yes, it should automatically convert any old databases to add the index, but rescue the error that's raised if the database already has the index. That seemed to be easier than checking for an index, or adding a more complicated versioned migration system.

Reply:
Love it ... thanks again!